### DISCUSSION TITLE
List of OpenSource dependencies report - FOSS report

### DISCUSSION THREAD

**Discussion by chris-reynier** [ImageMagick/ImageMagick#7697] [Help]
### ImageMagick version

7.x

### Operating system, version and so on

Windows / Linux

### Description

Hello,
Where can I find information about all the sub dependencies (name + version) used in a specific version of ImageMagick.
I have found old discussion threads (https://legacy.imagemagick.org/discourse-server/viewtopic.php?p=29943) but no clear place of this documentation
Thank you

**Comment by urban-warrior** [comment]
```
magick -list configure
magick -list format
magick -list delegate
```

**Comment by dlemstra** [comment]
The inno installer should create a file with the name `NOTICE.txt` in your installation folder with information about the versions and licenses of the project.
